### ISSUE TITLE
Duplicate genus Mimus

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#230]
https://www.catalogueoflife.org/data/search?facet=rank&facet=issue&facet=status&facet=nomStatus&facet=nameType&facet=field&limit=50&offset=0&q=Mimus&rank=genus&sortBy=taxonomic

Seems like a true homonym, one of which should be deprecated?

**Comment by DaveNicolson** [comment]
Yes: The beetle name Mimus Fåhraeus, 1871 is a junior homonym (of the bird name Mimus F. Boie, 1826), and it was replaced by the name Marvaldiella Alonso-Zarazaga & Lyal, 1999
http://weevil.info/sites/weevil.info/files/Alonso-Zarazaga%20%26%20Lyal,%201999_World%20Catalogue%20%28searchable%29.pdf
"Marvaldiella Alonso-Zarazaga & Lyal, nom. n. for Mimus Fahraeus, 1871 (non Boie, 1826) [cordially named after the promising Argentinian specialist in immature stages of weevils, Dra Adriana E. Marvaldi]"

**Comment by mdoering** [comment]
Hm, WTaxa causes quite some issues in COL. A large number of problems we spotted in the GBIF Backbone boiled down to WTaxa. @yroskov can we get a fresh copy from them? Ours is from 2016.

**Comment by mdoering** [comment]
Or is http://weevil.info maybe the better place for weevils? Sounds a bit like it, sth for the taxonomy group?

**Comment by yroskov** [comment]
A request for a new WTaxa copy has been sent to Miguel Alonso-Zarazaga.
